src/OdasStream/odas_stream.py

```diff
@@ -0,0 +1,96 @@
+import subprocess
+import json
+import time
+import math
+import os
+import sys
+from threading import Thread
+from PyQt5.QtCore import QObject, pyqtSignal
+
+
+class OdasStream(QObject):
+
+    signalOdasData = pyqtSignal(object)
```

Comment:
is it a global variable? if it is, you can use the tag "global" in front of "signalOdasData"